src/desktop/config.coffee

```diff
@@ -67,6 +67,8 @@ module.exports =
   MAX_POLLS_FOR_MAX_BIDS: 20
   MAX_SOCKETS: -1
   METAPHYSICS_ENDPOINT: null
+  METAPHYSICS_BLUE_ENDPOINT: null
+  METAPHYSICS_BLUE_LOAD_RATIO: 0
```

Comment:
How is this cast as an integer if passed in from the environment?

Reply:
It's just going to be set as an integer in the Heroku environment, I don't think we need to be super defensive here (since that can be set with the right type directly).

Reply:
All environment variables are strings, so you probably want to use `parseInt()` when retrieving it from the environment?

Reply:
@mzikherman how does Heroku set an env var as an integer?  In any case considering portability, I would definitely think `parseInt()` is required.

Reply:
I have no idea what kind of Heroku magic might be going on but on Kubernetes it's definitely going to come in as a string

Reply:
This must be handled somewhere else in the stack then

Reply:
Great point about being defensive, I definitely _have_ been bit in the past with strings vs ints vs booleans kinda thing with env vars.